services/traction/migrations/versions/ad0e08a1c94f_tenant_table.py

```diff
@@ -1,7 +1,7 @@
 """tenant table
```

Comment:
what wanted to re-format this file? it shouldn't matter, just odd that mine didn't do the same as yours.

Reply:
It was just a whitespace issue.

```
$ PYTHONPATH=. flake8 .
./config.py:18:121: E501 line too long (129 > 120 characters)
./migrations/versions/ad0e08a1c94f_tenant_table.py:4:9: W291 trailing whitespace
```

Reply:
Oh also black lol ...

```
$ black .
reformatted api/main.py
reformatted api/resources/ledger.py
reformatted migrations/versions/ad0e08a1c94f_tenant_table.py
reformatted migrations/env.py
reformatted api/acapy_utils.py
All done! ✨ 🍰 ✨
5 files reformatted, 6 files left unchanged.
```